### DISCUSSION TITLE
Catch all statments seem to be bugged

### DISCUSSION THREAD

**Discussion by apophis51** [vercel/next.js#59896] [Help]
### Summary

In the Next js documentation  you have an example of catch all statements here @ https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes#optional-catch-all-segments 

my issue is with [...catchall]  vs the [[...catchall]]  from what I see in the examples (and in testing)   [...catchall] already does what [[...catchall]] can do, and worse yet... my code crashes with anything with the [[...catchall]] format - has anyone had any success with the double [[   version? or can give me an example of working code?    according to the documentation "The difference between catch-all and optional catch-all segments is that with optional, the route without the parameter is also matched (/shop in the example above)."            but my issue seems to be that the non-optional catchall allready matches anything so whats going on here?

**Comment by apophis51** [comment]
solved it

**Reply by KempfCreative** [reply]
Can you explain what the fix or misunderstanding was?
